warehouse/utils/wsgi.py

```diff
@@ -16,10 +16,24 @@
 
 from typing import TYPE_CHECKING
 
+from sqlalchemy.exc import NoResultFound
+
+from warehouse.ip_addresses.models import IpAddress
+
 if TYPE_CHECKING:
     from pyramid.request import Request
 
 
+GEOIP_FIELDS = {
+    "continent_code": "CONTINENT",
+    "country_code": "COUNTRY_CODE",
+    "country_code3": "COUNTRY_CODE_3",
+    "country_name": "COUNTRY",
+    "region": "REGION",
+    "city": "CITY",
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏we've munged these when setting the headers from upstream, and we're now un-munging them. Since we haven't started to use them yet, maybe this is the time to fix them upstream, and thus preserve the same names throughout? https://github.com/pypi/infra/blob/f909316bf75e10ecafad8d558e0859c3e12d623a/terraform/warehouse/vcl/main.vcl#L186-L191

Reply:
hm? I don't quite follow what you're suggesting we do.

Reply:
Following this: https://github.com/pypi/infra/pull/133 If that's merged and deployed, then we update these inbounds to match the headers exactly.